lib/elastic_record/index.rb

```diff
@@ -40,9 +40,9 @@ class Index
     attr_accessor :load_from_source
 
     def initialize(models)
-      models = Array.wrap(models)
+      models = Array.wrap models
```

Comment:
Is removing the parentheses really necessary?

Reply:
Nah, that was a side effect of multiple iterations of this file and ultimately ending up back at the same functionality as before. I'll add the parentheses back in to reduce changes.